.devcontainer/post-create.sh

```diff
@@ -0,0 +1,13 @@
+RUBY_VERSION="$(cat .ruby-version | tr -d '\n')"
+
+cp .devcontainer/.env.codespaces .env
+
+# If the project's required ruby version changes from 3.2.2, this command
+# will download and compile the correct version, but it will take a long time.
+if [ "$RUBY_VERSION" != "3.2.2" ]; then
```

Comment:
Would be great if there was some way to have this tooling interact with the `.ruby-version` file in the root directory and auto-update. :pray: Ideally with a new image, but if not then via this script.

Reply:
Not sure what you mean. Are you concerned about the hard-coded `3.2.2`? Or are you wondering if `.ruby-version` could be updated automatically?

Reply:
The former. Basically it would be great if we could specify the version in one place and have it automatically apply to all the configuration relevant to it.

Reply:
I see and I agree, but not sure how to do that. 

The ruby version `3.2.2` comes from the image `mcr.microsoft.com/devcontainers/ruby:dev-3.2-buster`. We could use a different image and to get a different version of ruby. 

I think what you're hoping for is using the value in `.ruby-version` to choose which image we're running, but choosing the image happens before the filesystem is available, so we can't use that file to parameterize the `Dockerfile`. Also, the naming scheme is confusing and I don't know what the image names for ruby 3.2.4 or later will be. So, we're going to have to update the Dockerfile to choose them when they are available anyway.